src/ShopBundle/objects/db/TShopPaymentHandler/TShopPaymentHandlerPayPal_PayViaLink.class.php

```diff
@@ -192,7 +213,12 @@ public function ProcessIPNRequest($oOrder, $aURLParameter)
 
         if (0 != strcasecmp($sCurrency, $sPaymentCurrency)) {
             // invalid currency
-            $logger->error('PayPal IPN: invalid currency in request '.print_r($aURLParameter, true));
+            $logger->error(
+                'PayPal IPN: invalid currency in request.',
+                [
+                    'url_parameters' => $aURLParameter
```

Comment:
please add order id and ordernumber to the context

Reply:
ok